index.html

```diff
@@ -417,42 +384,57 @@
     };
 
     callback ThingEventListener = void (ConsumedThing thing);
+
+    enum ConsumedThingEvents {
+        "propertychanged",
+        "propertyremoved",
+        "propertyadded",
+        "actionchanged"
```

Comment:
I think 1. would be more efficient solution, because many TD changes could be expressed with one message, which also represents a TD change transaction, i.e. changes that needed to happen in the same time. Otherwise we cannot express this concept. So for the moment I will add the first event list.